### ISSUE TITLE
Difficulty re-ordering requests in folders

### ISSUE THREAD

**Issue by kbuzby** [a85/POSTMan-Chrome-Extension#3614]
## App Details:

```
Postman for Chrome
Version 5.3.0
win / x86-64
Chrome 61.0.3163.100
```

## Issue Report:
 1. Did you encounter this recently, or has this bug always been there: Encountered Today
 2. Expected behaviour: Request Should drop where the dividing line is
 3. Console logs (http://blog.getpostman.com/2014/01/27/enabling-chrome-developer-tools-inside-postman/ for the Chrome App, View->Toggle Dev Tools for the Mac app):
 4. Screenshots (if applicable):

Gif Recording:
![can t reorder in folder](https://user-images.githubusercontent.com/2800467/31360992-a5fe3c0e-ad16-11e7-99b4-795f249f4d30.gif)

I think this might be because "Flowsheet Rows" is a Save As copy of Successful - Minimum (as are the other "Missing Parameter" requests. So there could be some relation between the two that's getting stuck?

**Comment by prashantagarwal** [comment]
@kbuzby Thanks for reporting this. We will investigate the issue and keep you updated about this.

**Comment by YellowFish085** [comment]
The same issue is present in the Linux version of the app with some more problems.

```
Postman for Linux x64
Version 5.3.2
Ubuntu 16.04.3 LTS
```
![output2](https://user-images.githubusercontent.com/11556009/33487271-e79a988c-d6ac-11e7-933b-13fc3304eac4.gif)

As you can see in the gif above, in addition to the requests the issue is present for the folders too. Furthermore some folders are also deleted when moved and/or requests inside them seem to be deleted or created (according to the request count on the top) but I checked and they do not appear in the list. This seems to be random as sometimes folders move where I want them to move, sometimes they disappear, etc.

I must specify that a lot of the requests in the folders are duplicate of other requests, but since this particularity does not appear in the generated json when I export the collection I don't know if it matters.

**Comment by YellowFish085** [comment]
Update to my previous message.

I looked in the postman_collection.json file that is generated when you export a collection, and  it appears that some subdirectories didn't have the `_postman_isSubFolder` key.

I added it to all the subdirectories in the .json, removed the current collection from the app an imported the .json in the app, and for now I didn't encounter the problem described in this issue.

Could this be the origin of the issue described here?
